runtime/onert/api/include/nnfw.h

```diff
@@ -363,11 +363,14 @@ NNFW_STATUS nnfw_input_tensorinfo(nnfw_session *session, uint32_t index,
 /**
  * @brief     Get i-th output tensor info
  *
- * <p>Before {@link nnfw_prepare} is invoked, this function return tensor info in model,
- * so updated tensor info by {@link nnfw_apply_tensorinfo} is not returned.</p>
+ * <p>After {@link nnfw_load_model_from_file} and before {@link nnfw_prepare} is invoked, it returns
+ * tensor info in the model. The info called by {@link nnfw_set_input_tensorinfo} </p>
```

Comment:
Is `nnfw_set_input_tensorinfo` right?

Reply:
Yes, it will replaces `nnfw_apply_tensorinfo` so I changed it.

Reply:
`The info called by {@link nnfw_set_input_tensorinfo} </p>` seems redundant?

Reply:
I see.